core/src/main/scala/com/commercetools/queue/QueuePuller.scala

```diff
@@ -42,6 +42,7 @@ sealed trait QueuePuller[F[_], T] {
    * methods.
    */
   def pullBatch(batchSize: Int, waitingTime: FiniteDuration): F[Chunk[MessageContext[F, T]]]
+  def pullMessageBatch(batchSize: Int, waitingTime: FiniteDuration): F[MessageBatch[F, T]]
```

Comment:
Can you add some scala doc to the method explaining what the `MessageBatch` version is for?